onnxscript/function_libs/torch_lib/ops/common.py

```diff
@@ -0,0 +1,37 @@
+"""Common operators shared in the torchlib library."""
+
+import onnxscript
+import onnxscript.values
+from onnxscript import BOOL, INT64
+from onnxscript import opset18 as op
+from onnxscript.function_libs.torch_lib import _constants, tensor_typing
+
+DOMAIN = f"{_constants.DOMAIN}.common"
+
+
+class CommonOpset(onnxscript.values.Opset):
+    def __new__(cls):
+        return onnxscript.values.Opset.__new__(cls, DOMAIN, 1)
+
+    def Rank(self, input: tensor_typing.TTensor) -> INT64:
```

Comment:
Why do these need to be defined explicitly for `CommonOpset`, while for `torchlib` a single decorator is enough?

Reply:
This was a hack to allow both script and eager to work because `common.Rank` is currently rejected by the `script()` translator. I updated to alias `Rank` in the module level to circumvent this and removed this class.